compiler/cpp/src/thrift/generate/t_rs_generator.cc

```diff
@@ -907,16 +907,16 @@ void t_rs_generator::render_enum_impl(const string& enum_name) {
   // taking a reference to the enum
   f_gen_
     << indent()
-    << "pub fn write_to_out_protocol(self, o_prot: &mut dyn TOutputProtocol) -> thrift::Result<()> {"
+    << "fn write_to_out_protocol(&self, o_prot: &mut dyn TOutputProtocol) -> thrift::Result<()> {"
```

Comment:
Please remove the comment on line 984. As a result of your change the comment is no longer valid.